packages/analytics-remote-config/src/types.ts

```diff
@@ -15,7 +15,16 @@ export interface BaseRemoteConfigFetch<T> {
 }
 
 export interface RemoteConfigFetch<T> extends BaseRemoteConfigFetch<T> {
-  fetchTime: number;
+  metrics: Map<string, string | number>;
```

Comment:
Could you just add a comment stating that this will always be the last resolved value per key?

Reply:
I'd prefer have the comment at each metric level rather than make the statement for all metrics as it's possible that a new metric is for the fetcher creation. 

This is how the comments for each metric look like. It shows up when a metric is used. wdyt?
<img width="847" alt="image" src="https://github.com/user-attachments/assets/d72d7234-c0e5-47be-888a-7862308d0bac">

Reply:
Sounds good. Then can you include that as comment? (That the value is metric dependent)